### DISCUSSION TITLE
Please create connections counter

### DISCUSSION THREAD

**Discussion by orrans** [felladrin/linkedin-autoconnect-chrome-extension#23] [Ideas]
Please let us choose how much connection we want per session and after that it will stop
lets say I want only 27 connection so after 27 request the script will also stop
also please insert progress indicator
so after each click it will show the counter of how much connection its done until now (regardless the X connection we want) so it will show also in unlimited connection session and and also on the X connection session

**Comment by felladrin** [comment]
Thanks for the suggestion, @orrans. I'll try to include this feature on next release.

**Comment by felladrin** [comment]
This feature was implemented, and now you can set, in the Extension Option screen, the maximum number of connections for the session. It will stop auto-connecting as soon as it reaches the limit.
Please check the [latest release](https://github.com/felladrin/linkedin-autoconnect-chrome-extension/releases/latest) and let me know if it's what you expected.
